Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -370,6 +375,18 @@ def create_relationships(field: str, indicator: dict, resource: dict) -> list:
     return relationships
 
 
+def auto_detect_indicator_type_from_cs(value: str, type_: str) -> str | None:
+    '''
+    The function determines the type of indicator according to two cases::
+    1. In case the type is ip_address then the type is detected by auto_detect_indicator_type function (CSP).
+    2. In any other case, the type is converted by the table CROWDSTRIKE_TO_XSOAR_TYPES to a type of XSOAR.
+    '''
+    if type_ == 'ip_address':
+        return auto_detect_indicator_type(value)
```

Comment:
```suggestion
def auto_detect_indicator_type_from_cs(value: str, crowdstrike_resource_type: str) -> str | None:
    '''
    The function determines the type of indicator according to two cases::
    1. In case the type is ip_address then the type is detected by auto_detect_indicator_type function (CSP).
    2. In any other case, the type is converted by the table CROWDSTRIKE_TO_XSOAR_TYPES to a type of XSOAR.
    '''
    if crowdstrike_resource_type == 'ip_address':
        return auto_detect_indicator_type(value)
```